Test CAS005

chrisbrownie opened the pull request:
Added test CAS005 for DNS TTLs per #155

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/exchangeanalyzer/exchangeanalyzer/161)
<!-- Reviewable:end -->

cunninghamp commented:
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[Tests/CAS005.ps1, line 74 \[r1\]](https://reviewable.io:443/reviews/exchangeanalyzer/exchangeanalyzer/161#-KHO4rlX_zKOmJGNcDHB:-KHO4rlX_zKOmJGNcDHC:-1033712543) ([raw file](https://github.com/exchangeanalyzer/exchangeanalyzer/blob/40efb9b5659138b3e6cb0282075f1732c2b31e6e/Tests/CAS005.ps1#L74)):*
> ```PowerShell
>             ([System.Uri]$CAS.EWSExternal).Host,
>             ([System.Uri]$CAS.MAPIExternal).Host,
>             ([System.Uri]$CAS.AutoDSCP).Host
> ```

Should we bother trying to check the AutoD SCP externally?

---

*[Tests/CAS005.ps1, line 158 \[r1\]](https://reviewable.io:443/reviews/exchangeanalyzer/exchangeanalyzer/161#-KHO5c4Qpq3GTfuQNsbG:-KHO5c4Qpq3GTfuQNsbH:454020079) ([raw file](https://github.com/exchangeanalyzer/exchangeanalyzer/blob/40efb9b5659138b3e6cb0282075f1732c2b31e6e/Tests/CAS005.ps1#L158)):*
> ```PowerShell
>                 # doesn't exist or the resolver configuration is broken
>                 Write-Verbose "----- $TestID`: Unable to resolve namespace '$namespace' against external resolvers"
>                 $ErrorList += "Internal: '$namespace': (Unable to resolve)"
> ```

Should this error say "External:" instead?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/exchangeanalyzer/exchangeanalyzer/161)*
<!-- Sent from Reviewable.io -->

chrisbrownie commented:
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[Tests/CAS005.ps1, line 74 \[r1\]](https://reviewable.io:443/reviews/exchangeanalyzer/exchangeanalyzer/161#-KHO4rlX_zKOmJGNcDHB:-KHO8qg4Wt3hIvMH6Xsc:-1133344194) ([raw file](https://github.com/exchangeanalyzer/exchangeanalyzer/blob/40efb9b5659138b3e6cb0282075f1732c2b31e6e/Tests/CAS005.ps1#L74)):*
<details><summary><i>Previously, cunninghamp (Paul Cunningham) wrote…</i></summary>
> Should we bother trying to check the AutoD SCP externally?</details>
Removed this from the list of external namespaces

---

*[Tests/CAS005.ps1, line 158 \[r1\]](https://reviewable.io:443/reviews/exchangeanalyzer/exchangeanalyzer/161#-KHO5c4Qpq3GTfuQNsbG:-KHO8F5tNzv-JsAhGP2o:-497468049) ([raw file](https://github.com/exchangeanalyzer/exchangeanalyzer/blob/40efb9b5659138b3e6cb0282075f1732c2b31e6e/Tests/CAS005.ps1#L158)):*
<details><summary><i>Previously, cunninghamp (Paul Cunningham) wrote…</i></summary>
> Should this error say "External:" instead?</details>
Fixed

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/exchangeanalyzer/exchangeanalyzer/161)*
<!-- Sent from Reviewable.io -->

cunninghamp commented:
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/exchangeanalyzer/exchangeanalyzer/161)*
<!-- Sent from Reviewable.io -->
